packages/fxa-auth-server/bin/key_server.js

```diff
@@ -103,6 +109,26 @@ async function run(config) {
   /** @type {undefined | import('../lib/payments/stripe').StripeHelper} */
   let stripeHelper = undefined;
   if (config.subscriptions && config.subscriptions.stripeApiKey) {
+    if (
+      config.contentful &&
+      config.contentful.cdnApiUri &&
+      config.contentful.graphqlUrl &&
+      config.contentful.apiKey &&
+      config.contentful.spaceId &&
+      config.contentful.environment
```

Comment:
What happens if `config.contentful` doesn't have any values and CapabilityManager isn't added to the Container. Is CapabilityManager expected later in the code? Will it throw an error if not set?

Put another way,  does fxa-auth-server still startup? Does the getClients API endpoint still work?

Reply:
I've updated StripeHandler to return clients from Stripe if there is no CapabilityManager (as well as log to Sentry); I'll need to add a test for this.

fxa-auth-server should still startup; however, I don't believe getClients API endpoint would work if `config.contentful` does not have any values and CapabilityManager isn't added to Container.